packages/flutter/lib/src/cupertino/slider.dart

```diff
@@ -15,6 +15,19 @@ import 'colors.dart';
 import 'theme.dart';
 import 'thumb_painter.dart';
 
+typedef _SliderValueChanged = void Function(double value, bool isFastDrag)?;
+
+/// Defines the threshold for determining a "fast" slider drag.
+///
+/// Measured in slider extent per second.
+///
+/// For example, a threshold of 1.0 means that the user must drag with
+/// a velocity that will move the slider from start to end in 1 second.
+///
+/// A threshold of 0.5 means that the user must drag with a velocity
+/// that will move the slider 50% in 1 second.
+const double _kVelocityThreshold = 1.0;
```

Comment:
Can you add to these docs what device and what version of iOS you were using when you decided on this value? For example something like:

 > This value was estimated using a physical iPhone 16 running iOS 18.